_includes/docker_platform_matrix.md

```diff
@@ -1,12 +1,14 @@
+| [Debian]({{ install-prefix }}/docker-ce/debian.md)                         | [{{ green-check }}]({{ install-prefix }}/docker-ce/debian.md) | [{{ green-check }}]({{ install-prefix }}/docker-ce/debian.md) |                                                               |                                                                |                                                               |
+| [Fedora]({{ install-prefix }}/docker-ce/fedora.md)                         | [{{ green-check }}]({{ install-prefix }}/docker-ce/fedora.md) |                                                               |                                                               |                                                                |                                                               |
+| [Microsoft Windows Server 2016](/engine/installation/windows/docker-ee.md) |                                                               |                                                               |                                                               | [{{ green-check }}](/engine/installation/windows/docker-ee.md) |                                                               |
+| [Oracle Linux]({{ install-prefix }}/docker-ee/oracle.md)                   |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/oracle.md)  |                                                               |
+| [Red Hat Enterprise Linux]({{ install-prefix }}/docker-ee/rhel.md)         |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/rhel.md)    | [{{ green-check }}]({{ install-prefix }}/docker-ee/rhel.md)   |
+| [SUSE Linux Enterprise Server]({{ install-prefix }}/docker-ee/suse.md)     |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/suse.md)    | [{{ green-check }}]({{ install-prefix }}/docker-ee/suse.md)   |
+| [Ubuntu]({{ install-prefix }}/ubuntu.md)                                   | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ee/ubuntu.md)  | [{{ green-check }}]({{ install-prefix }}/docker-ee/ubuntu.md) |
+
+> **Note**: Support for IBM Z on Docker EE is limited to Docker EE Basic.
```

Comment:
I think you can have IBM Z nodes running as managers on a plain Docker swarm, but UCP supports IBM Z nodes running as workers (the same with Windows nodes too).
@friism can you confirm that this is the case?

If so, consider changing the note to something like "With Docker EE Standard and Advanced only supports IBM Z nodes running as workers."